conda-forge.yml

```diff
@@ -8,6 +8,8 @@ build_platform:
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
+# Use mambabuild to workaround https://github.com/conda-forge/conda-smithy/issues/1805
+conda_build_tool: mambabuild
```

Comment:
Not needed anymore after the last commit.

Reply:
Does that mean the `cross-python` stuff injects itself by overwriting the `host`'s `python` or something?

Reply:
Yes

Reply:
Ah, ok.
Would it be sensible to let it overwrite `${BUILD_PREFIX}/bin/pip` too (if it exists)?

(Yes, I know about the bonkers "we explicitly give you a `pip` executable, but you should actually only ever use `python -m pip`" thing people say and don't agree for obvious reasons.)

Reply:
Let's rather continue to discuss that at https://github.com/conda-forge/cross-python-feedstock/issues/76 .